Doc/using/cmdline.rst

```diff
@@ -994,3 +994,71 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. _using-on-finding-modules:
+
+Finding modules
+---------------
+
+A module search path is initialized when Python starts. This module search path
+may be accessed at :data:`sys.path`.
+
+The first entry in the module search path is the directory containing the input
+script (or the the current directory when no file is specified).
+
+On Windows a default Python installation adds the :file:`/Lib` directory and
+the :file:`/Lib/site-packages` directory to the module search path. These
+locations are relative to the :file:`python.exe` location. Standard modules
+are usually found in the Lib directory and third-party modules in the
+Lib/site-packages directory.
+
+On Unix the lib and lib/site-packages directories are added by looking for
+:file:`/exec_prefix/lib/pythonversion` , :file:`/exec_prefix/lib/pythonversion/site-packages` ,
+:file:`/prefix/lib/pythonversion` and :file:`/prefix/lib/pythonversion/site-packages` .
```

Comment:
In Linux, when Python 3.11 is built with separate locations for `--prefix` and `--exec-prefix`, I have the following entries in `sys.path` when running with `-S` (i.e. the site module is not imported):

```none
{script directory} or " "
{prefix}/lib/python311.zip
{prefix}/lib/python3.11
{exec_prefix}/lib/python3.11/lib-dynload
```

AFAIK, `"{exec_prefix}/lib/python3.11"` is never added to `sys.path`. Extension modules in the standard library are installed in "lib-dynload". The "python{X}{Y}.zip" file path for the zipped standard library is added whether or not it exists. 

Without `-S` (i.e. the site module is imported), the following site packages directories are added, if they exist:

```none
{userbase}/lib/python3.11/site-packages
{prefix}/lib/python3.11/site-packages
{exec_prefix}/lib/python3.11/site-packages
```

In POSIX, the default user base directory is `os.path.expanduser("~/.local")` -- except on macOS framework builds.

Reply:
Thank you for the detailed comments. I've edited the docs. I feel the details should be in the ```Using Python on Unix platforms``` area similar to the detailed Windows notes, so I'm trying to keep this initial overview as simple as possible which does bring the risk of not being completely comprehensive and accurate.